Homework1_Task2/BurrowWheelerTransformation.cs

```diff
@@ -0,0 +1,48 @@
+namespace Homework1_Task2;
+
+public class BurrowWheelerTransformation
+{
+    private static string GetNextStringRotation(string str)
+    {
+        var startElement = str[0];
+        return str.Remove(0, 1) + startElement;
+    }
+
+    public static KeyValuePair<string, int> DoBurrowsWheelerTransformation(string str)
+    {
+        var listOfStringRotations = new List<string>();
+        var newStr = string.Copy(str);
```

Comment:
```suggestion
        var newStr = str;
```
Зачем прям посимвольно копировать исходную строку?
(-0.1)